src/main/kotlin/com/demonwav/mcdev/translations/reference/contributors.kt

```diff
@@ -58,14 +58,10 @@ class JsonReferenceContributor : PsiReferenceContributor() {
                         return arrayOf()
                     }
                     val entry = element as? JsonProperty ?: return arrayOf()
-                    val range = entry.nameElement.textRangeInParent
-                    if (range.endOffset - 3 < range.startOffset) {
-                        return emptyArray()
-                    }
                     return arrayOf(
                         TranslationReference(
                             element,
-                            element.textRange,
+                            element.nameElement.textRangeInParent.shiftRight(1).grown(-2),
```

Comment:
Just glanced through this, not doing a full review yet.

But this took me a while to figure out what was going on, could you add a clarification comment here?